extensions/wikia/CommunityPage/CommunityPageSpecialController.class.php

```diff
@@ -57,50 +57,28 @@ public function index() {
 	 * @return array
 	 */
 	public function getTopContributorsData() {
-		$userContributionCount = $this->usersModel->getUserContributions( $this->getUser() );
-		$contributors = $this->usersModel->getTopContributors( 50 );
-		// get details for only 5 of the remaining contributors
-		$contributorDetails = $this->getContributorsDetails(
-			array_slice( $contributors, 0, self::TOP_CONTRIBUTORS_LIMIT )
+		$currentUserContributionCount = $this->usersModel->getUserContributions( $this->getUser() );
+		$topContributors = $this->usersModel->getTopContributors();
+		$top5ContributorsDetails = $this->getContributorsDetails(
```

Comment:
Maybe we can rethink this variable name because changing limit value in const will require changing var name